pulse_actions/handlers/talos.py

```diff
@@ -0,0 +1,79 @@
+"""
+This module is for the following use case:
+
+ - Talos jobs:
+
+    * Trigger talos jobs twice if they are from PGO build.
+"""
+import logging
+
+from pulse_actions.utils.misc import filter_invalid_builders
+
+from mozci import query_jobs
+from mozci.errors import PushlogError
+from mozci.mozci import (
+    trigger_job
```

Comment:
We normally use () when we want to import multiple pieces from a module.
Over here you only use trigger_job.